inst/examples/app.R

```diff
@@ -0,0 +1,38 @@
+library(shiny)
+library(shiny.i18n)
+
+# file with translations
+n18i <- i18n("translation.json")
+
+# change this to en
+n18i$set_translation_language("pl")
```

Comment:
very cool that you added a sample app!
Two general thoughts:

1. is it possible to have an app where each user chooses her own language? Imagine we have a select field with language -- could we use this solution to switch between languages for each user independently?

Or in other words, I'm wondering if we should have two separate steps:

```
translations <- i18n("translation.json")
translator <- translations$translator_for_language("pl")
```
this way we can run both steps before server function (as we have it now), but we could also run the second step inside the server function, if different users are supposed to have different languages set. What do you think?

2. I believe it's good to have a separate translation file for each language. This is how Django works (AFAIK) and I believe they have battle tested on huge number of websites and professional translators. Translations should be independent -- let's say we want to order translations to three different languages. We should be able to do this in parallel and simply receive a translations file from each translator. WDYT?